views/postgres/index.erb

```diff
@@ -37,13 +37,16 @@
         <tbody class="divide-y divide-gray-200 bg-white">
           <% @postgres_databases.each do |pg| %>
             <tr>
-              <td class="whitespace-nowrap py-4 pl-4 pr-3 text-sm font-medium text-gray-900 sm:pl-6" scope="row"><%= pg[:name] %></td>
+              <td class="whitespace-nowrap py-4 pl-4 pr-3 text-sm font-medium sm:pl-6" scope="row">
+                <a href="<%= @project_data[:path] %><%= pg[:path] %>" class="text-orange-600 hover:text-orange-700">
+                  <%= pg[:name] %>
+                </a>
+              </td>
               <td class="whitespace-nowrap px-3 py-4 text-sm text-gray-500"><%= pg[:location] %></td>
               <td class="whitespace-nowrap px-3 py-4 text-sm text-gray-500">
                 <%== render("components/pg_state_label", locals: { state: pg[:state] }) %>
               </td>
               <td class="relative whitespace-nowrap py-4 pl-3 pr-4 text-right text-sm font-medium sm:pr-6">
-                <a href="<%= @project_data[:path] %><%= pg[:path] %>" class="text-orange-600 hover:text-orange-700">Show</a>
               </td>
```

Comment:
Since the column is empty, you can remove this `<td></td>` tag. You also need to remove it from the header row.

Reply:
missed it, thanks!